content-service/server/database/search.js

```diff
@@ -9,22 +10,19 @@ module.exports = function(knex) {
     // ST_MakePoint is (x,y) so reverse conventional 'lat, long' to 'long, lat'
     const point = st.makePoint(long, lat);
 
-    return knex('beacon')
-      .select('channel_name', 'id', st.asGeoJSON('location'))
+    return utils.selectBeacons(knex)
       .where(st.dwithin('location', point, radius))
       .limit(100)
       .then((dbResponse) => {
-        return dbResponse.map((entry) => {
-          const parsedGeoJson = JSON.parse(entry.location);
-          return {
-            slug: numToShortId(entry.id),
-            channel_name: entry.channel_name,
-            location: {
-              latitude: parsedGeoJson.coordinates[1],
-              longitude: parsedGeoJson.coordinates[0],
-            }
-          };
-        });
+        return dbResponse
+          .map(utils.mapDatabaseResponseToApiResponse)
+          // For backwards compatibility duplicate some fields with their old
```

Comment:
@wilsonpage 

Would like to deprecate 'channel_name' and 'slug' from this response at some point. I've added them here for backwards compat.